.storybook/webpack.config.js

```diff
@@ -3,6 +3,7 @@ const path = require('path');
 const SCSS_PATH = path.resolve(__dirname, '../assets', 'scss');
 const CSS_PATH = path.resolve(__dirname, '../assets', 'css');
 const SRC_PATH = path.resolve(__dirname, '../src');
+const IMG_PATH = path.resolve(__dirname, '../assets', 'images');
```

Comment:
Are the images here just for the sake of the story? I'd recommend using placeholder images to avoid confusion about where to get app download images

Reply:
Images are only for the story. I can get them somewhere else